dask_ml/model_selection/_hyperband.py

```diff
@@ -1,5 +1,6 @@
 import math
```

Comment:
Thoughts on moving this filename to `hyperband.py`?  I'm not sure what the convention is behind private module names.

Reply:
The unstated convention convention is to mirror scikit-learn where applicable. They tend to use leading underscores (`model_selection/_split.py`)

Reply:
Ah, ok.  I retract my comment then